modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
+// TODO: we should add behavior information for other channel types, brokers, etc. other components if these are different to Kafka.
+
+== Apache Kafka channel event delivery
+
+If an event is successfully delivered to a Kafka channel receiver, the channel receiver will respond with a 202 code, which means that the event has been safely stored inside a Kafka topic and will not be lost. If the Kafka channel receiver responds with any other code, the event has not been safely stored, and steps must be taken by the user to resolve this issue.
```

Comment:
Replace every occurence of _code_ with _status code_, since it's the technical term